geode-core/src/main/java/org/apache/geode/internal/SharedLibrary.java

```diff
@@ -32,12 +27,6 @@
  */
 public class SharedLibrary {
```

Comment:
fix the class comment. It now has nothing to do with a shared library.

Reply:
Sounds fine. I was going to ask if this class should be set as deprecated, but renaming it sounds better.

Reply:
Would you accept `JVMSizeUtils`? This gives more info about what "sizes" are the class about.